class_ex/ex2/ex2.py

```diff
@@ -1,20 +1,27 @@
 class Person:
-    def __init__(self, name: str):
+    def __init__(self, name: str, age: int):
+        if type(name) is not str:
```

Comment:
```suggestion
        if not isinstance((name, str):
```
queda mas limpio usando la función isinstance()

Reply:
la he usado en todos ahora, creo que eta bien, miralo porque hay algunas que he metido dentro del if y no se si esta bien